ui/packages/consul-ui/app/components/consul/upstream-instance/list/index.hbs

```diff
@@ -63,7 +52,18 @@ as |mode|}}
               {{item.LocalBindSocketPath}}
             </dd>
           </dl>
-{{/let}}
+        {{#if item.LocalBindSocketMode}}
+          <dl class="local-bind-mode">
+            <dt>
+              <span>
+                Local bind mode
+              </span>
+            </dt>
+            <dd>
+              <span>mode:</span>{{or item.LocalBindSocketMode '-'}}
```

Comment:
According to the code here, we will never show a `-`?

Reply:
Oof, yea. I'll add another {{else}} block.

Reply:
I think here we can just remove the `if` entirely? I'm not 100% sure tho?